src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -926,7 +926,7 @@ object ReadInfo {
   /** Builds the [[ReadInfo]] by parsing a [[FastqRecord]]. */
   def apply(rec: FastqRecord): ReadInfo = this(rec.name, rec.comment.getOrElse(reject(rec.name)))
 
-  /** Builds the [[ReadInfo]] by parsing a [[DemuxRecord]]. */
+  /** Builds the [[ReadInfo]] by parsing a `DemuxRecord`. */
```

Comment:
I gave up, I couldn't get the reference to work

Reply:
I've also dealt with this in the past and sometimes it is not possible to fix.